### PR TITLE
[Detail] Movies list

### PR DESCRIPTION
**Pull request by OliRivDev**
Add movie section UIview and xib
Add Movie section to Details ViewController. Fix small issues
Add reference from new files to test projects

### CODE REVIEW

File: MarvelCharsApp/MarvelCharsApp/Views/DetailsViewController.swift [hunk 1]
```diff
@@ -24,7 +27,17 @@ class DetailsViewController: UIViewController {
         abilitiesView.addSubview(childView.view)
         abilitiesView.addConstrained(subview: childView.view)
         childView.didMove(toParent: self)
-    }
+        
+        
+        let movieCustomView = MovieSectionView(character: character!, frame: CGRect(x: 0, y: 0, width: 0, height: 0))
```
Comment (Fernando-Guerrero-jala) [inline MarvelCharsApp/MarvelCharsApp/Views/DetailsViewController.swift]: We should avoid force unwrapping with "!", use guard let instead

File: MarvelCharsApp/MarvelCharsApp/Views/DetailsViewController.swift [hunk 2]
```diff
@@ -12,10 +12,13 @@ class DetailsViewController: UIViewController {
...
     
+    @IBOutlet weak var stackView: UIStackView!
     var character: CharacterModel?
     
     override func viewDidLoad() {
         super.viewDidLoad()
+        // Make the top and bottom bar black
+        view.backgroundColor = .black
 
         let contentView = AbilitiesView(character: character!)
```
Comment (Fernando-Guerrero-jala) [inline MarvelCharsApp/MarvelCharsApp/Views/DetailsViewController.swift]: Force unwrapping here too

File: MarvelCharsApp/MarvelCharsApp/Views/Nibs/MovieSectionView.swift [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    
+    func loadViewFromNib() -> UIView? {
+        let nib = UINib(nibName: nibName, bundle: nil)
+        return nib.instantiate(withOwner: self, options: nil).first as? MovieSectionView
+    }
+}
+
+extension MovieSectionView: UICollectionViewDataSource {
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        let test = self.character?.movies.count ?? 0
```
Comment (Fernando-Guerrero-jala) [inline MarvelCharsApp/MarvelCharsApp/Views/Nibs/MovieSectionView.swift]: maybe variable shouldnt be called test?

File: MarvelCharsApp/MarvelCharsApp/Views/DetailsViewController.swift (hunk 1 above)
Comment (OliRivDev) [reply MarvelCharsApp/MarvelCharsApp/Views/DetailsViewController.swift]: Fixed

File: MarvelCharsApp/MarvelCharsApp/Views/DetailsViewController.swift (hunk 2 above)
Comment (OliRivDev) [reply MarvelCharsApp/MarvelCharsApp/Views/DetailsViewController.swift]: fixed

File: MarvelCharsApp/MarvelCharsApp/Views/Nibs/MovieSectionView.swift (hunk 3 above)
Comment (OliRivDev) [reply MarvelCharsApp/MarvelCharsApp/Views/Nibs/MovieSectionView.swift]: Changed

**Review by Fernando-Guerrero-jala — APPROVED** [review]

**Review by jmacboy — APPROVED** [review]
don't forget to merge with the last commit from develop first

**Pull request merged**